compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,36 @@
+# onecc-docker
+
+_onecc-docker_ is designed to improve user usability.
```

Comment:
Q) about word `designed`, in what persprective is about the design?

Reply:
@seanshpark 

Um, what word is more appropriate for this? `This tool improves user usability for ONE tools.` Is this sentence better than before? :)

Reply:
> This tool improves user usability for ONE tools.

as I know, this tool broadens ONE tools to be used in other platforms. correct me if this is different.
I'm not sure about the word `improves`. I think of `improve` would be provide more features or simple UI compared to existing tools.

Reply:
Then, how about this?

`_onecc-docker_ broadens ONE tools to be used in other platforms.`

Thank you for your comments. :)